rocketmq/ffi.py

```diff
@@ -117,6 +117,11 @@ class _CConsumeStatus(CtypesEnum):
     RECONSUME_LATER = 1
 
 
+class TransactionStatus(CtypesEnum):
```

Comment:
I'd call this `_CTransactionStatus` to discourage its direct usage and provider and simpler `IntEnum` for using in user provided callback, something like:

```python
class TranactionStatus(IntEnum):
    COMMIT = 0
    ROLLBACK = 1
    UNKNOWN = 2
```

Reply:
ok， I will polish it in next commit.